concent_api/core/tests/test_unit_model.py

```diff
@@ -162,3 +170,243 @@ def test_that_storing_subtask_with_task_to_compute_nested_in_another_messages_wi
                 force_get_task_result=force_get_task_result,
                 subtask_results_rejected=subtask_results_rejected,
             )
+
+
+class TestDepositAccountValidation(ConcentIntegrationTestCase):
```

Comment:
Please add test that removing `DepositClaim` do not remove related `DepositAccount`.

Reply:
Done.

Reply:
Instead of 
```
DepositClaim.objects.first().delete()
assert DepositAccount.objects.first() is not None
```

should be

```
self.deposit_claim.delete()
self.assertTrue(
    DepositAccount.objects.filter(pk=self.deposit_account.pk).exists()
)
```

Reply:
Done.